### ISSUE TITLE
Remove blue sparkle and coloring from Octoroks

### ISSUE THREAD

**Issue by BarrowRoll** [leoetlino/project-restoration#91]
Just as the sparkle points in the Great Bay Temple insulted players' intelligence, I believe that changing the Octoroks to blue and sparkling after getting the Ice Arrows makes it too obvious to freeze them to cross Ikana Canyon. It also just looks silly.

PS. Love this project, thank you for making Majora's Mask 3D the definitive version of this beautiful game!

**Comment by GeneraLight** [comment]
I would love for this change to be implemented. The blue Octoroks and their sparkles are out of place and distracting, in addition to being an insult to one's intelligence like you said. They weren't that way in the original.

**Comment by BarrowRoll** [comment]
I forgot to mention the single blue Chuchu that has the sparkle as well. Can't remember if anything else does as well, but ideally I'd like to see the sparkle removed from all areas of the game.

**Comment by GeneraLight** [comment]
Yeah. The sparkle is pretty, but it just makes the solution too easy to figure out.

**Comment by Fijiboy99** [comment]
I actually disagree, I would've likely never figured out you could freeze octorocks were it not for their different color scheme and sparkles. 

Plus, they're pretty.
